tests/unit/fs_metadata_log_test.cc

```diff
@@ -76,14 +76,21 @@ SEASTAR_THREAD_TEST_CASE(mock_metadata_to_disk_buffer_test) {
 
     log.bootstrap(0, 3, {3, 10}, 4, 0).get();
 
-    log.create_file("/test", file_permissions::default_dir_permissions).get();
+    log.create_directory("/test/", file_permissions::default_dir_permissions).get();
+    log.create_file("/test/test", file_permissions::default_dir_permissions).get();
 
     auto &created_buffers = mock_metadata_to_disk_buffer::created_buffers;
```

Comment:
I think it is a prefered way of making a reference. If you agree, please change in other places also.
```suggestion
    auto& created_buffers = mock_metadata_to_disk_buffer::created_buffers;
```